### DISCUSSION TITLE
Cosmos Cloud Manyfold3D Container broken on the Cosmos Marketplace

### DISCUSSION THREAD

**Discussion by zacgoodwin** [manyfold3d/manyfold#3283] [Support] [answered]
Howdy!

Do you all maintain the cosmos container or does the cosmos folk do? 

When installing using default settings I get this error: 

2024-12-07 20:10:09 Preparing database...
2024-12-07 20:10:10 bin/rails aborted!
2024-12-07 20:10:10 URI::InvalidURIError: the scheme postgresql does not accept registry part: 😡 (or bad hostname?) (URI::InvalidURIError)

Should be able to reproduce with a default cosmos install and a default manyfold install. I updated the DB URL with other strings I could think of and didn't have any luck. 

Where should I go for help?

**Comment by Floppy** [comment]
This is the place! Thanks for the logs and detail of the error on #2794, I'll try to work out what's going on!

**Comment by Floppy** [comment, marked as answer]
For future reference, the fix for this is to unset DATABASE_URL in cosmos (the template is wrong), and make sure that the password set for both manyfold and postgres are the same (the template also doesn't get that right). Should get some template fixes soon, but in the meantime, if anyone hits this, I hope this helps!

See #2794 for more background and detail.
